### ISSUE TITLE
Enhance and evangelise milestone-maintainers documentation and do training/retraining

### ISSUE THREAD

**Issue by LappleApple** [kubernetes/sig-release#1257]
During the Sept. 22 SIG Release meeting we noted that our docs for Milestone Maintainers (MMs)—see [here](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/release.md) and [here](https://github.com/kubernetes/sig-release/blob/master/release-team/README.md )—are not easily discoverable. The docs also need to be clearer and more consistent in their guidance to MMs. This item calls for three actions:

- [ ] Update the docs to defining the responsibilities and standards for milestone-ing an issue or PR
- [ ] Once docs are updated, make them more visible
- [ ] Supplement visibility efforts by proactively communicating the content of these docs and providing direction (training/retraining) to the broader Kubernetes org 

On the last point, SIGs have gotten some info via the Chairs&Leads forum, but we want to go further by helping MMs to perform and manage maintainer triage after code freeze.

Worth staying mindful of: [Add information about what qualifies for a milestone](https://github.com/kubernetes/sig-release/issues/320)

**Comment by mkorbi** [comment]
I would like to pick up this task

/remove-lifecycle stale
/assign @mkorbi

**Comment by mkorbi** [comment]
I opened two draft PRs on sig-release & community.

The idea is to move the milestone maintainer description to the community repository, also when the milestones maintainers are mainly steered through the release team, the pure process of maintaining milestones lays in the community.

Furthermore, I included a short section to the triage process in the community documentation.

sig-release PR: https://github.com/kubernetes/sig-release/pull/1459
community PR: https://github.com/kubernetes/community/pull/5484
